float32 distanceToTargetInMeters
float32 angleToTargetInDegrees